topics/eslint/index.md

```diff
@@ -0,0 +1,12 @@
+---
+created_by: Nicholas C. Zakas
+display_name: ESLint
+github_url: https://github.com/eslint/eslint
+logo: eslint.png
+related: eslint-plugin, eslint-config
```

Comment:
I think because neither of these topics have their own curated content yet, and they're very much dealing with ESLint, you could move them to `aliases` instead. That way, the curated content from this file will show up on the /topics/eslint-plugin and /topics/eslint-config pages.

You could also add linting, linter, and javascript to `related`.

Reply:
Done! ✅